pandas/core/arrays/base.py

```diff
@@ -661,6 +662,42 @@ def take(self, indices, allow_fill=False, fill_value=None):
         # pandas.api.extensions.take
         raise AbstractMethodError(self)
 
+    def where(self, cond, other):
```

Comment:
The other implementations of `where` (`DataFrame.where`, `Index.where`, etc.) have `other` default to NA.  Do we want to maintain that convention here too?